test/fixtures/discovery/host/expected_published_host_discovery.json

```diff
@@ -1,19 +1,24 @@
 {
-    "agent_id": "779cdd70-e9e2-58ca-b18a-bf3eb3f71244",
-    "discovery_type": "host_discovery",
-    "payload": {
-        "os_version": "15-SP2",
-        "ip_addresses": [
-            "10.1.1.4",
-            "10.1.1.5",
-            "10.1.1.6"
-        ],
-        "hostname": "thehostnamewherethediscoveryhappened",
-        "cpu_count": 2,
-        "socket_count": 1,
-        "total_memory_mb": 4096,
-        "agent_version": "trento-agent-version",
-        "installation_source": "Community",
-        "fully_qualified_domain_name": "com.example.trento.host"
-    }
-}
+  "agent_id": "779cdd70-e9e2-58ca-b18a-bf3eb3f71244",
+  "discovery_type": "host_discovery",
+  "payload": {
+    "os_version": "15-SP2",
+    "ip_addresses": [
+      "10.1.1.4",
+      "10.1.1.5",
+      "10.1.1.6"
+    ],
+    "netmasks": [
+      24,
```

Comment:
The only thing what concerns me is we must just think about using always the correct index in trento web :)

Reply:
As the length of ip addresses and netmasks is always the same, you can use the standard `zip` and `unzip` functions to combine them pretty easily

Reply:
Oh, today I learned about lodash [zipWith](https://lodash.com/docs/4.17.15#zipWith). Okay, this is cool!
Then never mind.